<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaAjustes</class>
 <widget class="QDialog" name="ventanaAjustes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajustes principales</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelAjustes">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;center&gt;Ajustes principales&lt;/center&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelCadacuanto">
       <property name="text">
        <string>Cada cuanto (segundos)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="textoCadacuanto"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Agregar Categor√≠a:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="textocategoria"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QListWidget" name="listacategorias"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="botonremover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="botonagregar">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textoCadacuanto</tabstop>
  <tabstop>textocategoria</tabstop>
  <tabstop>botonagregar</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>listacategorias</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ventanaAjustes</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ventanaAjustes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textocategoria</sender>
   <signal>returnPressed()</signal>
   <receiver>botonagregar</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
